patches/server/0211-Add-events-for-water-source-creation.patch

```diff
@@ -1,4 +1,4 @@
-From 62c738039a2cff268d2f5a9dbf2916741ff4ab1f Mon Sep 17 00:00:00 2001
+From bc723759409412b6e0964f958f8c5d5ba681bb10 Mon Sep 17 00:00:00 2001
```

Comment:
After I merge your other PR, these diffs should go away.

Reply:
These were coming up from `./sportpaper rebuild` regardless, would you rather I drop them in here and only resolve it with the other PR then?